background.ts

```diff
@@ -615,6 +614,29 @@ function handleFailure(payload: any) {
   }
   console.log(`Kubernetes was unable to start:`, payload);
   (async() => {
+    try {
+      const failureDetails = await k8smanager.getFailureDetails();
```

Comment:
This should really be part of the exception itself, rather than be attached to the backend.  That is, when we throw an exception from `childProcess.spawnFile` we know exactly what command failed, and what stream we wrote to.

Reply:
This only works for the `spawnFile` in `src/utils/childProcess.ts` but not the one in the std library, which we also call. And in a function like `lima.ts:lima()`, it has the command, but not the description (which mostly is based on the strings sent to the progress tracker.